src/polymer/polymer-element.ts

```diff
@@ -60,6 +60,10 @@ export interface Options {
     javascriptNode: estree.Expression | estree.SpreadElement,
     expression: LiteralValue
   }[];
+  listeners?: {
+    event: string,
+    handler: string
```

Comment:
What's the jump-to-definition use case? Jumping from the handler name to the actual method?

It would be good to validate the connection between the handler name and the corresponding method on the element, and if we did _that_ then I could see adding the source range of the method, but that's a big enough addition to this PR that it'd also be reasonable to punt that to another PR.

OTOH it would be pretty awesome.